packages/calcite-tokens/.gitignore

```diff
@@ -0,0 +1,40 @@
+$RECYCLE.BIN/
+*.bak
+*.d.ts
+*.lock
+*.log
+*.orig
+*.pem
+*.sublime-project
+*.sublime-workspace
+*.sw[mnpcod]
+*.tmp
+*.tmp.*
+*_BACKUP_*
+*_BASE_*
+*_LOCAL_*
+*_REMOTE_*
+*~
+.*cache*
+.DS_Store
+.DS_Store
+.Spotlight-V100
+.Trashes
+._*
+.env
+.idea/
+.sass-cache/
+.stencil/
+.turbo
+.versions/
+Dockerfile
+Thumbs.db
+UserInterfaceState.xcuserstate
+dist/
+ehthumbs.db
+log.txt
+logs
+node_modules/
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```

Comment:
Can you remove the entries that are duplicates of the root gitignore entries? It looks like all of them are redundant, unless I missed something.